cernopendata/modules/fixtures/data/records/jade-data.json

```diff
@@ -0,0 +1,78 @@
+[
+  {
+    "abstract": {
+      "description": "<p>JADE real data</p>",
+      "links": [
+        {
+          "description": "All JADE runs",
+          "recid": "FIXME"
+        }
+      ]
+    },
+    "accelerator": "DESY-PETRA",
+    "collaboration": {
+      "name": "JADE collaboration",
+      "recid": "26050"
+    },
+    "collections": [
+      "JADE-Primary-Datasets"
+    ],
+    "collision_information": {
+      "energy": "12GeV",
+      "type": "e+e-"
+    },
+    "date_created": [
+      "1986"
+    ],
+    "date_published": "2024",
+    "date_reprocessed": "1986",
+    "distribution": {
+      "formats": [
+        "raw"
+      ],
+      "number_events": 7017835,
+      "number_files": 2861,
+      "size": 2614994979
+    },
+    "doi": "10.7483/OPENDATA.9S5F.BY3B",
+    "experiment": [
+      "JADE"
+    ],
+    "files": [
+      {
+        "checksum": "adler32:44e5147f",
+        "size": 1575978,
+        "type": "index.json",
+        "uri": "root://eospublic.cern.ch//eos/opendata/jade/tapes_index.json"
+      }
+    ],
+    "keywords": [
+      "e+e- physics"
+    ],
+    "license": {
+      "attribution": "CC0"
+    },
+    "methodology": {
+      "description": "<p> During the experiment, the raw data was written to the tapes.</p>"
+    },
+    "publisher": "CERN Open Data Portal",
+    "recid": "FIXME",
+    "title": "JADE raw data",
+    "title_additional": "JADE raw data",
+    "type": {
+      "primary": "Dataset",
+      "secondary": [
+        "Collision"
+      ]
+    },
+    "usage": {
+      "description": "You can access these data through the EOS",
```

Comment:
The usage part should be mostly how one can use the data, i.e. some VM or Docker images, links to analysis examples, etc. (The fact that the data can be accessed through EOS is not giving enough details for the visitors to this page; they will already see download links, but they would not know what to do with the files.)

Reply:
The usage guide now poitns to @andriish's personal GitHub repository. Shall we point to the JADE guide present on the portal instead? We have pages like https://opendata-qa.cern.ch/docs/jade-getting-started that should provide basic information how to get started with analysing JADE data.  (And these docs pages can then point to @andriish's repository!)

Reply:
We now link the opendata site, but instead of `opendata-qa.cern.ch` we used `opendata.cern.ch`, is that fine?

Reply:
This will work only when the data will be released in production. For now it's better to use relative links, e.g. "/docs/jade-getting-started", so that people could peruse things easily on QA. I'll change this during merge.